### ISSUE TITLE
Package as Flatpak for elementary OS 6

### ISSUE THREAD

**Issue by danrabbit** [TenderOwl/Norka#49]
As you might have heard, AppCenter is going Flatpak! For elementary OS 6, we will not longer be accepting Debian packaging, so your app will need to be updated.

We have documentation on our developer guide for writing your Flatpak manifest here: https://docs.elementary.io/develop/writing-apps/our-first-app/packaging#flatpak-manifest

Also, in case you missed it, elementary OS 6 beta 2 is [now available](https://builds.elementary.io). This is a great time to get your app packaged up as Flatpak and test that everything is working before the final release. AppCenter submission for OS 6 will be open soon, so don't delay :)

**Comment by JohnDumpling** [comment]
Norka is still not available as a Curated app in AppCenter.

**Comment by amka** [comment]
@danrabbit I tried but failed with error at flatpak action with the same result: https://github.com/TenderOwl/Norka/runs/3489483621?check_suite_focus=true#step:4:280
```
../gtksourceview/gtksourceinit.c:25:10: fatal error: gtksourceview-gresources.h: No such file or directory
```

Actually, I thought it could be easy enough to look at what you did in [Code's](/elementary/code) repo and copy flatpak instructions but it is not.

Maybe you could look at the error and give me an idea?

**Comment by amka** [comment]
Ah, I found the solution! Problem fixed in GtkSource within version 4.8.2.
